src/js/components/editor/Sign.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+    <span class="sign"
+      @click="onClick"
+      :class="signClasses"
+    >
+      <span v-if="showReconstructedStart" class="reconstruction-start">[</span>
+      <span class="sign-value" :class='{whitespace: isWhitespace}'>{{isWhitespace ? '  ' : sign.sign}}</span>
```

Comment:
Maybe we offer the functionality to show a gray dot or some other character for space (like Word does with "show formatting")  that can be helpful for finding double spaces.

Reply:
Yes, I like that. Adding a card in our project!